### PR TITLE
Make thingtalk dialogs translatable

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Previously hard-coded EN texts were used, now moved to the translation files.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -151,7 +151,7 @@
       },
       "cover": {
         "position": "Position",
-        "tilt_position": "Tilt position"
+        "tilt_position": "Tilt"
```
Comment (bramkragten) [inline src/translations/en.json:154]: Why this change?

**Review by bramkragten — APPROVED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (spacegaier) [reply src/translations/en.json:154]: Ah, that snuck in from some other changes for the cover cards I am working on. It sounds weird to me to have "Position" and "Tilt position" since with the first slider we have a real position in terms of "up/down" or "left/right" (e.g. gates). Combining then the words "tilt" and "position" for the second slider feels weird, since now suddenly "position" refers to a different concept (namely the tilt).

Either way, I will remove it from this PR since unrelated topic.
